### PR TITLE
fix: correct height of suggested repositories for mobile screen

### LINKED ISSUE #1552: Bug: suggested repository on create insight page is squashed
### Describe the bug

In the create new insight page on mobile device the suggested repository looks squashed
**Here's a screenshot**
Screenshot_20230813-132125

### Steps to reproduce

1. Go to https://insights.opensauced.pizza/hub/insights/new
2. Scroll down see error 

### Browsers

Chrome

### Additional context (Is this in dev or production?)

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by himanshumalviya15**
fix #1552

## Description

fixed the height of suggested repositories for mobile screen

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents
Fixes #1552

### Old:
260308513-5589bd51-d3db-430b-bdd0-c1d5c7437c77

### New: 
image

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

File: components/molecules/RepoCardProfile/repo-card-profile.tsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ const RepoCardProfile = ({ avatar, orgName, repoName, prCount, issueCount, class
       <div className="h-max group-hover:border rounded-lg">
         <Avatar initials="BD" size={40} avatarURL={avatar} />
       </div>
-      <div className="flex flex-col gap-1 h-[46px]">
+      <div className="flex flex-col gap-1 h-[77px]">
```
Comment (bdougie) [inline components/molecules/RepoCardProfile/repo-card-profile.tsx:20]: The problem is that we are using px and not the tailwind design system. This change fixes mobile but breaks desktop.

Screen Shot 2023-08-15 at 4 00 39 PM

**Review by bdougie — CHANGES_REQUESTED** [review]
Let's fix this `px` by making mobile only changes instead.

**Comment by himanshumalviya15** [comment]
@bdougie Please review latest changes 
image

File: components/molecules/RepoCardProfile/repo-card-profile.tsx [hunk 2]
```diff
@@ -17,7 +17,7 @@ const RepoCardProfile = ({ avatar, orgName, repoName, prCount, issueCount, class
       <div className="h-max group-hover:border rounded-lg">
         <Avatar initials="BD" size={40} avatarURL={avatar} />
       </div>
-      <div className="flex flex-col gap-1 h-[46px]">
+      <div className="flex flex-col gap-1 sm:h-[60px]">
```
Comment (bdougie) [inline components/molecules/RepoCardProfile/repo-card-profile.tsx:20]: This is not correct either. You are now removing the height and still adjusting the pixels. I appreciate the attempt but this is now removing height css from desktop screens all together and only applying mobile.

**Comment by bdougie** [comment]
@himanshumalviya15 we need to discuss the approach you are taking in solving this. Please provide suggestions approach the fix in a way that will not cause future bugs in the UI. 

Considering asking the [discord](https://discord.gg/opensauced) for feedback. 

@open-sauced/triage team ping for help.

**Comment by himanshumalviya15** [comment]
@bdougie can we have some like this

 sm:h-[60px] md:h-[66px] lg:h-[70px] xl:h-[72px]
 
 It will cover whole ranges of sizes

**Comment by bdougie** [comment]
> @bdougie can we have some like this
> 
> sm:h-[60px] md:h-[66px] lg:h-[70px] xl:h-[72px]
> 
> It will cover whole ranges of sizes

No. Because the following reasons

1. This is not respecting the rest of the tailwind design system. We cannot continue to add `px` as band-aids. Anything that perpetuates the use of px is not a solution. 
2. The issue is the mobile screen. Solve the mobile view without breaking desktop.

**Review by a0m0rajab — CHANGES_REQUESTED** [review]
Thanks for the effort, I think that changing this to tailwind design system might solve the issue, you can check: https://tailwindcss.com/docs/height to see what is the best option and apply it.

**Comment by takanome-dev** [comment]
IMO, changing the height for mobile only will break the alignment of the repo card and the add to page button. But @himanshumalviya15 if you manage to have something that looks good, propose it 🙂

## Suggestions

- Use `gap` instead of `height`



But this push the text on smaller screens.

- Change the design of the cards on mobile

| Design 1 | Design 2 |
| ------ | ------ |
|  |  |

The design 2 looks good :smiley: 

> **Note**
>
> The space on the right(on mobile) is caused by the `HubLayout`. Trying to remove it may break other pages. Maybe we can tackle it in another issue if really necessary

File: components/molecules/RepoCardProfile/repo-card-profile.tsx (hunk 2 above)
Comment (NsdHSO) [reply components/molecules/RepoCardProfile/repo-card-profile.tsx:20]: could you use `h-16` -> `64px` ???

**Comment by himanshumalviya15** [comment]
@bdougie @takanome-dev Please have a look I corrected my PR according to design provide

**Comment by bdougie** [comment]
Can you adjust the width of the button to be less than 100%

## @takanome-dev example
image

## design
Screen Shot 2023-08-17 at 5 09 33 AM

**Comment by himanshumalviya15** [comment]
@bdougie Now it is pixel perfect
image

File: components/molecules/RepoCardProfile/repo-card-profile.tsx (hunk 2 above)
Comment (bdougie) [reply components/molecules/RepoCardProfile/repo-card-profile.tsx:20]: Visually this looks great now. This comment needs to be addressed. https://github.com/open-sauced/insights/issues/1522

**Pull request closed without merging**